From c63a0041af095ce1be591d27401e7252a9f52ebd Mon Sep 17 00:00:00 2001
From: Robert Schwebel <r.schwebel@pengutronix.de>
Date: Thu, 14 Jan 2010 09:30:56 +0100
Subject: [PATCH 1/2] libnl-1.1-undefined-ulong-max.diff

In file included from addr.c:28:
../include/netlink-local.h: In function '__str2type':In file included from attr.c:12:
../include/netlink-local.h: In function '__str2type':
../include/netlink-local.h:218: error: 'ULONG_MAX' undeclared (first use in this function)

This needs limits.h

Patch is already in the upstream git at
http://git.kernel.org/?p=libs/netlink/libnl.git

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
---
 include/netlink-local.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/netlink-local.h b/include/netlink-local.h
index 3c50ea0..d946934 100644
--- a/include/netlink-local.h
+++ b/include/netlink-local.h
@@ -26,6 +26,7 @@
 #include <sys/socket.h>
 #include <inttypes.h>
 #include <assert.h>
+#include <limits.h>
 
 #include <arpa/inet.h>
 #include <netdb.h>
-- 
1.6.6

